blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,244 @@
+
+- **Extended Support Timeline:** 24 months of full support from GA date (compared to standard 12-15 months), including security patches and critical bug fixes
+- **Premium Tier Requirement:** Available as part of the Premium tier, which includes additional enterprise features and comes with associated costs
+- **Same Kubernetes Experience:** Identical functionality to standard AKS with proven, stable Kubernetes versions
+- **Backwards Compatibility:** Strong commitment to API stability and workload compatibility within LTS versions
+- **Integrated Azure Services:** Compatibility with most existing Azure services and AKS ecosystem features (see [LTS limitations](https://learn.microsoft.com/en-us/azure/aks/long-term-support#unsupported-add-ons-and-features) for current exceptions as we work towards 100% coverage)
+
+## Enterprise Benefits at a Glance
+
+**Reduced Operational Overhead**
+- Fewer required upgrades mean less time spent on cluster maintenance
+- Predictable planning cycles aligned with enterprise budgeting and resource allocation
+- Reduced testing and validation overhead for workload compatibility
+
+**Enhanced Stability**
+- Production-tested Kubernetes versions with proven track record
+- Minimized risk of introducing breaking changes or regressions
+- Focus on reliability over cutting-edge features
+
+**Compliance and Governance**
+- Extended support timelines that meet enterprise compliance requirements
+- Clear end-of-life planning with advance notice for migration planning
+- Integration with Azure Policy and governance frameworks
+
+## Technical Implementation
+
+**Universal LTS Coverage - A Major Expansion:** Previously, AKS LTS was available only for select Kubernetes versions, typically spaced several versions apart. **This announcement represents a fundamental shift: AKS LTS now extends to every currently supported Kubernetes version in AKS.** 
+This means instead of waiting for specific "LTS-designated" versions, enterprises can choose long-term support for any supported Kubernetes version that meets their application requirements.
+
+**Beyond Community Support:** When the Kubernetes community support ends (typically 12-14 months after a version's GA), AKS LTS continues providing comprehensive support for 24 months from the original GA date. **Crucially, once the community stops supplying patches after the official version support end-of-life, AKS continues to provide patches and CVE fixes** for all supported components.
+
+This extended support is comprehensive and includes:
+
+- **Core Kubernetes components:** API server, etcd, kubelet, kube-proxy, and all core Kubernetes functionality
+- **AKS-managed add-ons, extensions, and AKS features:** (see [**AKS integrations**](https://learn.microsoft.com/azure/aks/integrations) for the complete list), **including networking, monitoring, and security components**, with the exception of Istio which is coming soon as mentioned in the Looking Ahead section.
```

Comment:
can we verify the limitations list? let's discuss this internally

Reply:
Yes its verified to the best of my knowledge